parsing_tests/test_refwriter.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+from pyingest.parsers.iop import IOPJATSParser
+from pyingest.serializers.classic import Tagged
+from pyingest.serializers.refwriter import ReferenceWriter
+from glob import glob
+
+
+outfile = 'iop_test.tag'
+
+journal_ISSN = {
+               '1538-3881': 'AJ',
+               '0004-6256': 'AJ',
+               '1475-7516': 'JCAP',
+               '0004-637X': 'ApJ',
+               '1538-4357': 'ApJ',
+               '0067-0049': 'ApJS',
+               '2041-8205': 'ApJ',
+               '1538-4357': 'ApJL',
+               '2041-8205': 'ApJL',
+               '2515-5172': 'RNAAS'
+               }
+
+parser = IOPJATSParser()
+
+basedir = '/proj/ads/articles/sources/STACKS/'
```

Comment:
I would point basedir to the fulltext root /proj/ads/articles/fulltext/sources (the STACKS directory really is only a staging area). The stucture of /proj/ads/articles/fulltext/sources is always predictable.

Reply:
I made this change, thanks!